### DISCUSSION TITLE
Cannot read properties of null (reading 'path')

### DISCUSSION THREAD

**Discussion by Lukas-3d** [SilentVoid13/Templater#1158] [Help]
Hello everyone,

when I was looking for a solution to automize the creation of my meeting minutes in obsidian, I stumbled across [this article](https://www.thoughtasylum.com/2023/02/26/my-obsidian-generic-meeting-template/#generating-the-link).
I followed it step-by-step and consulted as well the [older one](https://www.thoughtasylum.com/2022/03/29/auto-link-and-generate-page-in-obsidian/) it is referencing to.
Unfortunately I still cannot get it running.

I have the buildPageAndLink.js in the dedicated folder for JS scripts. 
In the templater settings I referenced this folder for user script functions.
the file ending is .js
In the templater folder for templates I have the Meeting Minutes Template with the following code
`<%* tR += await tp.user.buildPageAndLink(tp, "M.Meeting - General (Template)", "Change the Title", "", true); %>`

When I now call the Template I get the following error in Obsidians console:

templater-obsidian:61 Templater Error: Template parsing error, aborting. 
 Cannot read properties of null (reading 'path')

The "" should mean that the Meeting Minute Note shall be placed in my root vault, if I got it right. I tried several other folders as well but I received the same error every time.

My search in the forum was not successful. I would be grateful if you could point me towards my mistake.

**Comment by Zachatoo** [comment]
> The "" should mean that the Meeting Minute Note shall be placed in my root vault

I think that is incorrect. I ran the following code in the dev console in Obsidian and here are the results.

<img width="368" alt="image" src="https://github.com/SilentVoid13/Templater/assets/6936914/9551e97b-4db4-4f01-8d33-b35cdb5f25f7">

Pretty sure you want to use `"/"` if you want the root.

I don't know if that will fix everything for you, but hopefully that helps.
